enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/analytics.ts

```diff
@@ -58,10 +58,12 @@ function reportAnalyticsOnce(activeEmbeds: Set<MetabaseEmbedElement>) {
   if (!isAnalyticsReported) {
     isAnalyticsReported = true;
     const [firstEmbed] = activeEmbeds;
-    firstEmbed.sendMessage(
-      "metabase.embed.reportAnalytics",
-      createEmbeddedAnalyticsJsUsage(activeEmbeds),
-    );
+    firstEmbed.addEventListener("ready", () => {
+      firstEmbed.sendMessage("metabase.embed.reportAnalytics", {
+        usageAnalytics: createEmbeddedAnalyticsJsUsage(activeEmbeds),
+        embedHostUrl: window.location.href,
```

Comment:
We need to send the host location from here, otherwise, accessing `window.parent.location` isn't always available based on the security settings.